### ISSUE TITLE
Scala 3

### ISSUE THREAD

**Issue by TobiasRoland** [scanamo/scanamo#1465]
Hi there!

Scala 3 is finally pretty solid, so it might be worth considering publishing scanamo for Scala 3! I didn't spot an issue for it, so I figured I would create one for discussion/blockers/etc.

**Comment by vasiliybondarenko** [comment]
How to can I use `scanamo` with Scala 3 right now?

**Comment by i10416** [comment]
If you are Scala-CLI user

```scala
//> using scala "3.2.2"
//> using lib "org.scanamo::scanamo:1.0.0-M25"

import org.scanamo._
import org.scanamo.generic.auto._
import software.amazon.awssdk.services.dynamodb.DynamoDbClient

case class Foo(i:Int,j:String)

@main def run = 
    val scanamo = Scanamo(DynamoDbClient.create())
    val record = Foo(1,"")
    val db = Table[Foo]("tablename")
    val program = for {
    _ <- db.put(record)
    } yield ()
    scanamo.exec(program)

```

**Comment by rtyley** [comment]
This issue was closed by https://github.com/scanamo/scanamo/pull/1577 / https://github.com/scanamo/scanamo/pull/1596 , first released as [v1.0.0-M23](https://github.com/scanamo/scanamo/releases/tag/v1.0.0-M23).
